docs/targets/twilio.md

```diff
@@ -1,119 +1,50 @@
+## Deploying an Instance of the Target
 
-On the other hand `defaultPhoneTo` wont be usually informed unless you want all messages to address the same phone number by default.
+Open the Bridge creation screen and add a Target of type `Twilio`.
 
-Both configurations can be overrided at every Cloud Event message received at the Target.
+![Adding a Twilio Target](../images/twilio-target/create-bridge-1.png)
 
-Refer to [Twilio docs for number formating](https://www.twilio.com/docs/lookup/tutorials/validation-and-formatting?code-sample=code-lookup-with-international-formatted-number).
+In the Target creation form, give a name to the event Target and add the following information:
 
-### Status
+* **Default source phone number**: Defines a default sending phone. This is not mandatory; however, it is usually configured to the [phone number][pn] purchased at Twilio.
+* **Default destination**: If informed, all messages will default to this phone number.
+* **SID Secret**: Reference to a [TriggerMesh secret][tm-secret] containing a SID token for authenticating requests.
+* **Token Secret**: Reference to a [TriggerMesh secret][tm-secret] containing an API token for authenticating requests.
 
-TwilioTarget requires two Secrets to be provided for SID and Token, once they are present it will create a Knative Service. Controller logs and events can provide detailed information about the process. A Status summary is added to the TwilioTarget object informing of the all conditions that the target needs.
+Both the **Default source phone number** and **Default destination** configurations can be overrided by any [CloudEvent][ce] message received at the Target.
 
-When ready the `status.address.url` will point to the internal point where Cloud Events should be sent.
+For more information about using Twilio, please refer to the [Twilio documentation][docs]
 
-### Twilio Target as an event Sink
+![Twilio Target form](../images/twilio-target/create-bridge-2.png)
 
-Twilio Target is addressable, which means you can use it as a Sink for Knative components.
+After submitting the bridge, and allowing some configuration time, a green check mark on the main _Bridges_ page indicates that the bridge was successfully created.
 
-```yaml
-apiVersion: eventing.knative.dev/v1beta1
-kind: Trigger
-metadata:
-  name: <TRIGGER-NAME>
-spec:
-  broker: <BROKER-NAME>
-  filter:
-    attributes:
-      type: <MESSAGE-TYPES-TWILIO-FORMATTED>
-  subscriber:
-    ref:
-      apiVersion: targets.triggermesh.io/v1alpha1
-      kind: TwilioTarget
-      name: <TARGET-NAME>
-```
+![Bridge status](../images/bridge-status-green.png)
 
-### Sending SMS to a Twilio Target
+## Event Types
 
-Twilio Target expect a JSON payload at the CloudEvent that includes:
+The Twilio event Target can consume events of any type; however, it expects a [JSON][[ce-jsonformat]] payload at the [CloudEvent][ce] with the following properties:
```

Comment:
check sentence. "payload at the..."

Reply:
Replace the semicolon with a comma. Also, change `it` to more explicitly reference `the Target`.

Reply:
is the ';'  not correct?  
ref_> https://web.sonoma.edu/users/f/farahman/subpages/utilities/however.pdf